CHANGELOG.md

```diff
@@ -18,7 +18,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ### Features
 
 - Prism now has got a static example fallback in case the `dynamic` flag is not enabled #347
-- Prism can now validate servers #351
```

Comment:
Oops!

Reply:
This is correct, the server validation is not in alpha 15, but in alpha 16